docs/quick-start.md

```diff
@@ -0,0 +1,92 @@
+# Quick Start Guide
+
+## Overview
```

Comment:
Is this mac only? mac and linux?

Maybe also mention that this guide is optimized for automatticians.

Reply:
+1, we should be clear who the intended audience is.

Except for the `--cask` I pointed out elsewhere, I think everything in here will work for Linux too. The only A12 thing I notice is the bit about Jurassic Tube, which does call that out and mention alternatives.